src/hr_alg3.rs

```diff
@@ -2,8 +2,8 @@
 
 use ringbuffer::{ConstGenericRingBuffer, RingBuffer};
 
-const CRAZY_HI: i32 =3000;
-const CRAZY_LO: i32 =-1000;
+const CRAZY_HI: u32 =3000;
+const CRAZY_LO: u32 =1000;
```

Comment:
This is confusing, but I don't see how to avoid it. You now have to know to subtract the LO value because the sign was removed